src/docfx/docset/Docset.cs

```diff
@@ -225,15 +225,24 @@ Repository GetRepositoryInternal(string fullPath)
             }
         }
 
-        private static IReadOnlyDictionary<string, string> NormalizeRoutes(Dictionary<string, string> routes)
+        private static IReadOnlyDictionary<string, string> NormalizeRoutes(Dictionary<string, string> routes, string sourceBasePath)
```

Comment:
consider changing the logic inside `ApplyRoutes` over here, that is where all the logic is applied, this method just does path normalization.

Reply:
if I move this logic into ApplyRoutes, this logic will run everytime the apply routes is called
this method is to normalize the routes, which is exactly what I am doing, right? 

---
In reply to: [284527991](https://github.com/dotnet/docfx/pull/4481#discussion_r284527991) [](ancestors = 284527991)